ci-operator/config/openshift-knative/eventing-kafka/openshift-knative-eventing-kafka-release-next__47.yaml

```diff
@@ -41,6 +41,22 @@ images:
       - destination_dir: .
         source_path: /go/bin/distributed_dispatcher
   to: knative-eventing-kafka-distributed-dispatcher
+- dockerfile_path: openshift/ci-operator/knative-images/mtcontroller/Dockerfile
+  from: base
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/mtcontroller
+  to: knative-eventing-kafka-mtcontroller
+- dockerfile_path: openshift/ci-operator/knative-images/mtreceive_adapter/Dockerfile
+  from: base
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/mtreceive_adapter
+  to: knative-eventing-kafka-mtreceive-adapter
```

Comment:
These are not about 4.8, right?

Reply:
yes we can build them for 4.7 too (have not been updating that section that often)